### PR TITLE
add page profil page candidate

### PR DESCRIPTION
**Pull request by bastien-fournier**

### CODE REVIEW

File: client/src/components/AvatarUser/AvatarUser.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import avatarUser from "../../assets/logo/avatarUser.png";
+
+export default function PAvatarUser() {
```
Comment (wilderLPM) [inline client/src/components/AvatarUser/AvatarUser.jsx:3]: Small typo on the function name

**Review by wilderLPM — COMMENTED** [review]
Small fix needed

**Review by wilderLPM — APPROVED** [review]
All good !

**Review by HazeFury — APPROVED** [review]
ok 👍

**Pull request merged**